for_developers/cli_guide.md

```diff
@@ -313,3 +313,35 @@ overrides:
       init_args:
         patience: 3
 ```
+
+## How to use OTX Workspace
+
+If we run a typical Training example, we'll have a folder like the one below as output.
+
+```console
+otx-workspace/
+    .cache/                       # As of the last training, we have a .cache with path information for config and checkpoint.
```

Comment:
```suggestion
    .latest/                       # As of the last training, we have a .latest with path information for config and checkpoint.
```
I'm feeling that `.cache` is not an appropriate naming for it. Consider another name more suitable, e.g., `.latest_conf.`.

Reply:
Thank you for the great suggestion. - https://github.com/openvinotoolkit/training_extensions/pull/2934/commits/6b560edf42ebdfdbf9bc3268df6adcb375a47f07